applications/compareRestart.hpp

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
what's the purpose of this header file? i don't see a reason for us to expose application internals to the world.